### ISSUE TITLE
affichage des maisons au delà de 6

### ISSUE THREAD

**Issue by kaori01** [JulienNigon/SpellcasterUniversity-Bug-Report#876]
lorsque l'on a "trop" de maisons, les dernières ne sont pas totalement visible dans l'écran de recrutement... 

![image](https://user-images.githubusercontent.com/60629516/75658204-1f515700-5c68-11ea-94c5-29013950c4e4.png)

**Comment by kaori01** [comment]
une précision : par contre elles sont  quand même interactives, à présent ;)

**Comment by Neelan22** [comment]
oui, on a corrigé ce point mais il faut qu'on rajoute un "slider" pour les faire défiler et qu'elles rentrent meiux dans l'interface
